client/my-sites/site-settings/theme-setup-dialog/index.jsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import PulsingDot from 'components/pulsing-dot';
+import { getSelectedSite } from 'state/ui/selectors';
+import { toggleDialog, runThemeSetup } from 'state/ui/theme-setup/actions';
+
+class ThemeSetupDialog extends React.Component {
+	renderButtons( { runThemeSetup, site, isActive, result, translate } ) {
+		const keepContent = {
+			action: 'keep-content',
+			label: translate( 'Set Up Your Theme' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Setting up your theme...' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).